app/scripts/views/mixins/signin-mixin.js

```diff
@@ -91,6 +91,17 @@ define(function (require, exports, module) {
         // If it is, the error should be displayed on this screen
         // and the user shouldn't even have the chance to continue.
 
+        // get unblock code here and try it
+        const storedCodes = JSON.parse(window.sessionStorage.getItem('unblockCode')) || {};
```

Comment:
Why use sessionStorage to store the unblockCode? Seems like it'd be simpler just to [allow a new field on the Account model](https://github.com/mozilla/fxa-content-server/blob/1acf7aeb09a2c5f46f8dc5b2ce2601042c8021e3/app/scripts/models/account.js#L61).